src/zocalo/cli/configure_rabbitmq.py

```diff
@@ -100,21 +101,27 @@ def update_config(
 
 
 def get_binding_specs(group: Dict) -> List[BindingSpec]:
-    source = group.get("bindings", "")
+    sources = group.get("bindings", [""])
     vhost = group.get("vhost", "/")
-    bspecs = [
-        BindingSpec(
-            vhost=vhost,
-            source=source,
-            destination=name,
-            destination_type="q",
-            routing_key=name,
-            arguments={},
-            properties_key=name,
+    return list(
+        itertools.chain(
+            *[
+                [
+                    BindingSpec(
+                        vhost=vhost,
+                        source=source,
+                        destination=name,
+                        destination_type="q",
+                        routing_key=name,
+                        arguments={},
+                        properties_key=name,
+                    )
+                    for source in sources
+                ]
+                for name in group["names"]
+            ]
         )
-        for name in group["names"]
-    ]
-    return bspecs
+    )
```

Comment:
Is this equivalent to
```python
    return [
                    BindingSpec(
                        vhost=vhost,
                        source=source,
                        destination=name,
                        destination_type="q",
                        routing_key=name,
                        arguments={},
                        properties_key=name,
                    )
                    for source in sources
                    for name in group["names"]
          ]
````

Reply:
I did try something like this, but must have got the nested list comprehension syntax wrong, as the code above does appear to work.